memory-viz/src/automate.ts

```diff
@@ -102,7 +103,8 @@ function drawAutomatedStackFrames(
 } {
     for (const req_prop of REQUIRED_DISPLAY_PROPERTIES) {
         if (!configuration.hasOwnProperty(req_prop)) {
-            configuration[req_prop] = config.obj_x_padding;
+            (configuration as Record<keyof DisplaySettings, any>)[req_prop] =
```

Comment:
Hi @hemmatio, regarding your comment about this: yeah, it seems that adding a `string` value to `DisplaySettings` messes with the type inference here.

Rather than specify a type for `configuration`, you can instead a type for `configuration[req_prop]` as `number | undefined` (since the keys in `REQUIRED_DISPLAY_PROPERTIES` all refer to numeric values).